### ISSUE TITLE
Diamond Run (diamond) yields "'[Dipswitch] Energy Loss not found error'"...

### ISSUE THREAD

**Issue by roslof** [finalburnneo/FBNeo#1548]
[Tested on Raspberry Pi with head at commit 1b1c67c]

Received libretro pop-up from lr-fbneo reading:
```
Default value for '[Dipswitch] Energy Loss' not found, it's a bug, please report it
```

**Comment by dinkc64** [comment]
Thanks for the bug report, the problem is fixed (and should make it to the lr branch soon)

best regards,
- dink

**Comment by roslof** [comment]
Confirmed fixed with build at head c7498a3.
Have a great day, Dink
